pyproject.toml

```diff
@@ -14,12 +14,10 @@ commands =
 
 [tool.coverage.run]
 branch = true
-data_file = 'tests/.coverage'
+data_file = "tests/.coverage"
```

Comment:
I am just wondering if is there a specific reason for changing it from single to double quotation mark?